core/domain/rights_manager.py

```diff
@@ -222,6 +289,20 @@ def update_activity_first_published_msec(
 
 
 def create_new_exploration_rights(exploration_id, committer_id):
+    """Creates new exploration rights and save to the datastore.
```

Comment:
Creates a new exploration rights object, and saves it to the datastore.

(Don't worry if you need this initial docstring to go to 2+ lines -- that's totally OK. I know it's a convention elsewhere, but we gave up worrying about that constraint some time ago.)